core/commands/root.go

```diff
@@ -94,6 +94,7 @@ var rootSubcommands = map[string]*cmds.Command{
 	"mount":     MountCmd,
 	"name":      NameCmd,
 	"object":    ObjectCmd,
+	"patch":     PatchCmd,
```

Comment:
@jbenet had mentioned that he thinks it would be good to have under the global namespace. Also, if we have to type object in there, the commands become absurdly long